scene/gui/control.h

```diff
@@ -149,6 +149,12 @@ class Control : public CanvasItem {
 		TEXT_DIRECTION_INHERITED = TextServer::DIRECTION_INHERITED,
 	};
 
+	enum AutoTranslateMode {
+		AUTO_TRANSLATE_MODE_INHERIT, // Same as parent `Control`/`Window`.
+		AUTO_TRANSLATE_MODE_ALWAYS, // Always auto translate.
+		// AUTO_TRANSLATE_MODE_DISABLED, // Never auto translate. Commented until compatibility breakage is allowed, where it will replace the `auto_translate` property.
```

Comment:
```suggestion
		//AUTO_TRANSLATE_MODE_DISABLED, // Never auto translate. Commented until compatibility breakage is allowed, where it will replace the `auto_translate` property.
```